.ci/scripts/validate_commit_message_custom.py

```diff
@@ -88,6 +88,13 @@ def check_commit(commit_sha):
             LOG.error(f"Commit {commit_sha[:8]} has no issue attached")
             return False
 
+    commit_url = f'https://api.github.com/repos/ansible/galaxy_ng/commits/{commit_sha}'
```

Comment:
we're also running this script in the UI .. the repo url should probably ultimately come from `GITHUB_REPOSITORY`

Reply:
@himdel good catch!